src/php/tests/interop/interop_client.php

```diff
@@ -332,10 +332,11 @@ function timeoutOnSleepingServer($stub) {
   $opts['update_metadata'] = $auth->getUpdateMetadataFunc();
 }
 
-$stub = new grpc\testing\TestServiceClient(
-    new Grpc\BaseStub(
-        $server_address,
-        $opts));
+$internal_stub = new Grpc\BaseStub($server_address, $opts);
+hardAssert($internal_stub->getTarget() == $server_address,
```

Comment:
Got it. Fixed. Changed all those to just `is_string` checks. Thanks for the feedback.